frontend/src/components/dashboard/demo/create-demo-modal/CreateDemoModal.tsx

```diff
@@ -79,6 +76,15 @@ const CreateDemoModal = ({
     }
   }
 
+  function showDemoVideoModal() {
+    setModalTitle(null)
+    setModalContent(
```

Comment:
I actually don't understand how this doesn't cause a circular dependency problem. It works, but why?